config/routes.rb

```diff
@@ -169,6 +176,10 @@
 
     resource :session, only: %i[create destroy] do
       post 'mfa_create', to: 'sessions#mfa_create', as: :mfa_create
+      collection do
+        get :webauthn_authentication_options, constraints: { format: :json }, defaults: { format: :json }
```

Comment:
Consider altering the `:webauthn_authentication_options` route to use `post` instead of `get` to take advantage of Rails' CSRF protection.

Reply:
Was coded as a `GET` because not a state-changing action.